region/maridia/outer.json

```diff
@@ -1374,17 +1374,39 @@
                 {
                   "name": "Base",
                   "notable": false,
-                  "requires": ["Gravity"]
+                  "requires": [
+                    "Gravity",
+                    {
+                      "or": [
+                        "canWalljump",
+                        "SpaceJump",
+                        "canIBJ",
+                        {"and": [
+                            "HiJump",
+                            "SpeedBooster"
+                          ]},
+                        "canSpringBallJump"
+                      ]
+                    }
+                  ]
                 },
                 {
-                  "name": "Suitless",
+                  "name": "Springball to the Fish",
                   "notable": false,
                   "requires": [
                     "canSuitlessMaridia",
                     "canUseFrozenEnemies",
                     "HiJump",
                     "canSpringBallJumpMidAir"
                   ]
+                },
+                {
+                  "name": "Double Springball Jump",
+                  "notable": true,
```

Comment:
Does this require anything more than what canDoubleSpringBallJumpMidAir would imply? If not, notable should be false.

Reply:
Agreed, should be false.